src/main/java/org/cascadebot/cascadebot/commands/moderation/PurgeLinkSubCommand.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.moderation;
+
+import net.dv8tion.jda.api.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.*;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.PurgeUtils;
+
+
+public class PurgeLinkSubCommand implements ISubCommand {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length < 1) {
+            context.getUIMessaging().replyUsage();
+            return;
+        }
+
+        if (!context.isArgInteger(1)) {
```

Comment:
Your checking for an args length of 1 yet your getting the second arg

Reply:
Resolved @weeryan17